src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionSettings.cs

```diff
@@ -12,6 +12,8 @@ internal sealed class HttpConnectionSettings
     {
         private const string Http2SupportEnvironmentVariableSettingName = "DOTNET_SYSTEM_NET_HTTP_SOCKETSHTTPHANDLER_HTTP2SUPPORT";
         private const string Http2SupportAppCtxSettingName = "System.Net.Http.SocketsHttpHandler.Http2Support";
+        private const string Http2PlainSupportEnvironmentVariableSettingName = "DOTNET_SYSTEM_NET_HTTP_SOCKETSHTTPHANDLER_HTTP2PLAINSUPPORT";
```

Comment:
How about `UNENCRYPTED` instead of `PLAIN`, since that's the lingo the http2 FAQ uses?

Reply:
UNENCRYPTED sounds good. One more option we may consider is something with h2c  as that how RFC refers to protocol without encryption.

Reply:
I'm fine with that, too.

Reply:
I decided to go with your original proposal.  I was hard for me to see difference between Http2 and Http2c